Sources/ParseSwift/Storage/ParseStorage.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  File.swift
+//  
+//
+//  Created by Pranjal Satija on 7/19/20.
+//
+
+// MARK: ParseStorage
+public struct ParseStorage {
+    public static var shared = ParseStorage()
+
+    private var backingStore: PrimitiveObjectStore!
+
+    mutating func use(_ store: PrimitiveObjectStore) {
+        self.backingStore = store
+    }
+
+    private mutating func requireBackingStore() {
+        guard backingStore != nil else {
```

Comment:
I don't think we should create the inMemory store for them, if they don't supply one, we should fatalError and tell them to supply one. We can give them the option to use the one you created, but setting it up for them enables developers to not pay attention and then complain about the storage using memory

Reply:
I'll update this. We can move it to the `initialize` call that users have to make when providing an app ID and stuff.